stack/dashboard/deb/builder.sh

```diff
@@ -36,6 +36,10 @@ if [ "${repository}" ];then
     valid_url='(https?|ftp|file)://[-[:alnum:]\+&@#/%?=~_|!:,.;]*[-[:alnum:]\+&@#/%=~_|]'
     if [[ $repository =~ $valid_url ]];then
         url="${repository}"
+        if ! curl --output /dev/null --silent --head --fail "${url}"; then
+        echo "The given URL to download the Wazuh plugin zip does not exist: ${url}"
```

Comment:
Lines inside if block should have visible indentation

Reply:
Done: bf0df521